### ISSUE TITLE
Is aria-label the correct attribute to use?

### ISSUE THREAD

**Issue by jeremiak** [CalMatters/scrolly-photos#1]
The component currently takes the alt text from the children images and adds them to the `<div>` with an [`aria-label`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) attribute, but should it instead be [`aria-description`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-description) or [`aria-describedby`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby)?

From the MDN docs:
> The aria-description attribute is similar to [aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) in that both provide a text string to associate with the element, but a label should be short and concise, while the description can be longer as it is intended to provide more context and information.
